### ISSUE TITLE
LIDC数据集

### ISSUE THREAD

**Issue by Pang-b0** [Yancccccc/SMR-UNet#6]
您好，我看到您的论文中说了LIDC-IDRI数据集是预处理过的，请问在下载链接中有说明如何预处理后得到该处理后的数据集吗？

**Comment by Yancccccc** [comment]
没有，直接下载另外一篇论文的

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年11月20日 17:58 |
| 收件人 | Yancccccc/SMR-UNet ***@***.***> |
| 抄送人 | Subscribed ***@***.***> |
| 主题 | [Yancccccc/SMR-UNet] LIDC数据集 (Issue #6) |

您好，我看到您的论文中说了LIDC-IDRI数据集是预处理过的，请问在下载链接中有说明如何预处理后得到该处理后的数据集吗？

**Comment by Pang-b0** [comment]
> 没有，直接下载另外一篇论文的
> […](#)
> ---- 回复的原邮件 ---- | 发件人 | ***@***.***> | | 发送日期 | 2024年11月20日 17:58 | | 收件人 | Yancccccc/SMR-UNet ***@***.***> | | 抄送人 | Subscribed ***@***.***> | | 主题 | [Yancccccc/SMR-UNet] LIDC数据集 (Issue #6) | 您好，我看到您的论文中说了LIDC-IDRI数据集是预处理过的，请问在下载链接中有说明如何预处理后得到该处理后的数据集吗？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

您好 我在测试您的的best model时，在test似乎遇到了一个模型不匹配的问题，请问是SMR-UNet代码错误还是您提供的模型文件错了

**Comment by Pang-b0** [comment]
![image](https://github.com/user-attachments/assets/ca7e60ab-1b7c-4858-bedf-fc3d2a8150d0)
下边是我的测试代码
colormap = [[0,0,0], [1,1,1]]
dataset = LIDC_IDRI(dataset_location = 'data/') # change path
test_sampler = torch.load('sampler/test_sampler.pth') # change path
test_loader = DataLoader(dataset, batch_size=1, sampler=test_sampler,shuffle=False)
net = SMRUNet.Unet(1,1)
net.load_state_dict(torch.load('checpoint/SMRUNet/bestmodel_LIDC_no_lr_step.pth'))
net.cuda()
